basis-universal/Cargo.toml

```diff
@@ -15,6 +15,8 @@ categories = ["game-development", "graphics", "api-bindings", "compression", "en
 basis-universal-sys = { version = "0.1.0", path = "../basis-universal-sys" }
 lazy_static = "1.4.0"
 bitflags = "1.2.1"
+basis-universal-wasm = { git = "https://github.com/expenses/basis-universal-rs-wasm" }
```

Comment:
Can/should these dependencies be optional?

Reply:
Also we'd need basis-universal-wasm to be published with a license compatible with this crate (no copyleft, or copyleft-ish licenses like MPL). I noticed there are some other new licenses probably from js-sys as a dependency. The ones I saw are fine but will need to be allow-listed in deny.toml

Reply:
> Can/should these dependencies be optional?

Yep, I'll do that.